### PR TITLE
docs: Update Installation doc

### PR DESCRIPTION
**Pull request by TheBrunoLopes**
## Pull request description 
Updates the Installation docs to make it more simple

### CODE REVIEW

File: docs/installing.md [hunk 1]
```diff
@@ -238,10 +210,12 @@ The following Helm defaults are used in the `testkube` chart:
...
 <https://github.com/bitnami/charts/tree/master/bitnami/mongodb#parameters>
 
-## **Uninstall Testkube**
-
-Uninstall Testkube using the purge command integrated into the Testkube plugin.
-
+## **Remove Testkube Server Components**
+Using helm
+```sh
+helm delete testkube
```
[the remark this answers is not in the document]
Comment (TheBrunoLopes) [reply docs/installing.md:216]: I think testkube is installed by default in the "testkube" namespace right ?
Should it be "-n testkube" ?

File: docs/installing.md [hunk 2]
```diff
@@ -1,119 +1,100 @@
-# Installation
+# Installation Steps
 
-To get Testkube up and running:
+To get Testkube up and running you need to:
 
-1. Install the kubectl testkube plugin.
-2. Install Testkube in your cluster.
-3. Configure Testkube's Dashboard UI Ingress for your ingress-controller, if needed.
+1. Install the Testkube CLI
```
Comment (jfermi) [inline docs/installing.md:5]:
```suggestion
1. Install the Testkube CLI.
```

File: docs/installing.md [hunk 3]
```diff
@@ -1,119 +1,100 @@
...
+### **Homebrew (MacOS)**
 
+You can install Testkube from Homebrew:
 ```sh
 brew install testkube
 ```
-
-Or we're building our own Homebrew tap for each release, so you can easily install latest Testkube version.
-
+Or directly from our tap. The Homebrew mantainers take a few days/or week to approve each one of our releases so you can use our tap to make sure you always have the most recent release.
```
Comment (jfermi) [inline docs/installing.md:27]:
```suggestion
Or directly from our tap. The Homebrew mantainers take a few days or a week to approve each one of our releases so you can use our tap to make sure you always have the most recent release.
```

File: docs/installing.md [hunk 4]
```diff
@@ -1,119 +1,100 @@
...
-```sh
-brew update 
-brew upgrade testkube
-```
+### **Chocolatey (Windows)**
 
-### **Installing on Windows**
-
-You can easily install latest Testkube version from our own chocolatey repository .
+You can use [Chocolatey](https://chocolatey.org/install) to install  version from our own chocolatey repository .
```
Comment (jfermi) [inline docs/installing.md:36]:
```suggestion
You can use [Chocolatey](https://chocolatey.org/install) to install a version from our own chocolatey repository .
```

File: docs/installing.md [hunk 5]
```diff
@@ -1,119 +1,100 @@
...
-choco upgrade testkube
-```
-
-### **Installing on Linux with apt**
-
-You can use apt to install Testkube.
+#### **APT (Debian/Ubuntu)**
 
-Download our public GPG key, and add them to the trusted keys:
+1. Download our public GPG key, and add them to the trusted keys:
```
Comment (jfermi) [inline docs/installing.md:45]:
```suggestion
1. Download our public GPG key, and add it to the trusted keys:
```

File: docs/installing.md [hunk 6]
```diff
@@ -1,119 +1,100 @@
...
 
-### **Installing on Linux or MacOS with Install Script**
+## **Manual Download**
 
-To install on Linux or MacOs, run
-
-```sh
-bash < <(curl -sSLf https://kubeshop.github.io/testkube/install.sh )
-```
+If you don't want automatic scripts or package managers you can always do a manual install:
```
Comment (jfermi) [inline docs/installing.md:65]:
```suggestion
If you don't want to use automatic scripts or package managers you can always do a manual install:
```

File: docs/installing.md [hunk 7]
```diff
@@ -1,119 +1,100 @@
...
 
-### **Alternative Installation Method - Manual**
-
-If you don't like automatic scripts you can always do a manual install:
-
-1. Download binary with the version of your choice (the most recent one is recommended).
-2. Unpack it (tar -zxvf testkube_0.6.5_Linux_arm64.tar.gz).
+1. Download binary with the version of your choice and platform of your choice [here](https://github.com/kubeshop/testkube/releases)
```
Comment (jfermi) [inline docs/installing.md:67]:
```suggestion
1. Download the binary with the version of your choice and platform of your choice [here](https://github.com/kubeshop/testkube/releases)
```

File: docs/installing.md [hunk 8]
```diff
@@ -1,119 +1,100 @@
...
+2. Unpack it. For example, (tar -zxvf testkube_0.6.5_Linux_arm64.tar.gz) for Linux
```
Comment (jfermi) [inline docs/installing.md:68]:
```suggestion
2. Unpack it. For example,  in Linux use (tar -zxvf testkube_0.6.5_Linux_arm64.tar.gz).
```

File: docs/installing.md [hunk 9]
```diff
@@ -1,119 +1,100 @@
...
+For Windows, you will need to unpack the binary and add it to `%PATH%` as well.
 
-We have plans to build installers for the most popular Operating Systems and system distros [#161](https://github.com/kubeshop/testkube/issues/161).
+If you use a package manager that we don't support, please let us know here [#161](https://github.com/kubeshop/testkube/issues/161).
 
-## **Install Testkube Components in Your Cluster**
 
-[Helm](https://helm.sh) must be installed to use charts.  
-Please refer to  Helm's [documentation](https://helm.sh/docs) to get started.
+# **2. Testkube Server Componets**
```
Comment (jfermi) [inline docs/installing.md:76]:
```suggestion
# **2. Testkube Server Components**
```

File: docs/installing.md [hunk 10]
```diff
@@ -1,119 +1,100 @@
...
+To deploy Testkube to your K8s cluster you will need the following packages installed:
+- [Kubectl docs](https://kubernetes.io/docs/tasks/tools/) 
+- [Helm docs](https://helm.sh/docs/intro/install/#through-package-managers)
 
-The Testkube kubectl plugin provides an init command to install Testkube in your cluster. Note: you must have helm installed
 
+## Using Testkube's CLI to deploy the Server Components
```
Comment (jfermi) [inline docs/installing.md:82]:
```suggestion
## Using Testkube's CLI to Deploy the Server Components
```

File: docs/installing.md [hunk 11]
```diff
@@ -1,119 +1,100 @@
...
+The Testkube CLI provides an command to deploy the Testkube server components to your cluster easly.
```
Comment (jfermi) [inline docs/installing.md:83]:
```suggestion
The Testkube CLI provides a command to easily deploy the Testkube server components to your cluster.
```

File: docs/installing.md [hunk 12]
```diff
@@ -1,119 +1,100 @@
...
 Run:
-
 ```shell
-kubectl testkube init
+testkube init
 ```
+note: you must have your KUBECONFIG ponting to the desired location of the installation.
```
Comment (jfermi) [inline docs/installing.md:88]:
```suggestion
Note: you must have your KUBECONFIG pointing to the desired location of the installation.
```

File: docs/installing.md [hunk 13]
```diff
@@ -163,13 +144,8 @@ replicaset.apps/testkube-minio-testkube-64cd475b94   1         1         1
...
 By default Testkube is installed in the `testkube` namespace.
 
-### **Manual Testkube Helm Charts Installation**
-
-[Helm](https://helm.sh) must be installed to use charts.  
-Please refer to  Helm's [documentation](https://helm.sh/docs) to get started.
-
-Once Helm has been set up correctly, add the Kubeshop Helm repository as follows:
-
+## **Using HELM to deploy the Server Components**
```
Comment (jfermi) [inline docs/installing.md:147]:
```suggestion
## **Using HELM to Deploy the Server Components**
```

**Review by jfermi — COMMENTED** [review]
A few editing suggestions.

**Pull request merged**